### PR TITLE
Add 2022q2 Azure report

### PR DESCRIPTION
**Pull request by lwhsu**

### CODE REVIEW

File: 2022q2/azure.adoc [hunk 1]
```diff
@@ -0,0 +1,39 @@
...
+** The taks will be benefited by merging of ZFS support of makefs(8) and release(7)
+*** https://reviews.freebsd.org/D23334
+*** https://reviews.freebsd.org/D34426
+*** https://reviews.freebsd.org/D35248
+* Build and publish gen2 vm image to Azure Marketplace
+** Blocked by https://bugs.freebsd.org/264267
+
+Above tasks are sponsored by The FreeBSD Foundation, with resources provided by Microsoft.
+
+Wei Hu and his colleagues in Microsoft are working on several tasks sponsored by Microsoft:
```
Comment (pauamma) [inline 2022q2/azure.adoc:28]: Noting that sponsoring is usually at the end of the section. See report-sample.adoc.

**Review by pauamma — CHANGES_REQUESTED** [review]

File: 2022q2/azure.adoc [hunk 2]
```diff
@@ -0,0 +1,39 @@
...
+Contact: Wei Hu <whu@FreeBSD.org> +
+Contact: Li-Wen Hsu <lwhsu@FreeBSD.org> +
+
+The link:https://azuremarketplace.microsoft.com/marketplace/apps/thefreebsdfoundation.freebsd-13_1[13.1-RELEASE image on Azure Marketplace] has been published.
+
+Work in progress tasks:
+
+* Automatizing the image building and publishing process
+* Build and publish ZFS-based image to Azure Marketplace
+** The taks will be benefited by merging of ZFS support of makefs(8) and release(7)
```
[the remark this answers is not in the document]
Comment (lwhsu) [reply 2022q2/azure.adoc:19]: I think I'll leave this for the quarterly report maintainers. (and thanks!)

File: 2022q2/azure.adoc (hunk 1 above)
Comment (lwhsu) [reply 2022q2/azure.adoc]: Yeah, but those items are sponsored by different entities so I want to be more explicit.

**Comment by lwhsu** [comment]
Please feel free to modify more before merging. Thanks!

File: 2022q2/azure.adoc (hunk 2 above)
[the remark this answers is not in the document]
Comment (pauamma) [reply 2022q2/azure.adoc:19]: > I think I'll leave this for the quarterly report maintainers. (and thanks!)

Changed it as lsalvadore confirmed in IRC it should work.

File: 2022q2/azure.adoc (hunk 1 above)
Comment (pauamma) [reply 2022q2/azure.adoc]: > Yeah, but those items are sponsored by different entities so I want to be more explicit.

Left it and added a summary in the usual location.

**Comment by pauamma** [comment]
> Please feel free to modify more before merging. Thanks!

I only saw this after merging and additional edits. Can you confirm they work for you?

**Comment by lwhsu** [comment]
They are fine. Thanks!

**Pull request merged**